core/server/models/plugins/pagination.js

```diff
@@ -144,11 +144,18 @@ pagination = function pagination(bookshelf) {
             // Get the table name and idAttribute for this model
             var tableName = _.result(this.constructor.prototype, 'tableName'),
                 idAttribute = _.result(this.constructor.prototype, 'idAttribute'),
-                self = this,
+                self = this;
+
+            let countPromise;
+            if (options.transacting) {
+                countPromise = this.query().clone().transacting(options.transacting).select(
```

Comment:
Nice, I tried finding this in the docs but failed. I'd attempted to use `(options.transacting || bookshelf.knex).raw()` but that wasn't working

Reply:
Didn't find this in the docs either... was more of a trial/error/reading the source approach :joy: